src/features/contact/contactmoment/service.ts

```diff
@@ -85,6 +87,41 @@ const postContactmoment = (data: Contactmoment): Promise<Response> => {
     body: JSON.stringify(data),
   });
 };
+
+export const saveKlantContact = async (
+  data: KlantContact,
+): Promise<SaveKlantContactResponseModel> => {
+  const response = await postKlantContact(data);
+  const responseBody = await response.json();
+
+  if (response.ok) {
+    return { data: responseBody };
+  }
+
+  if (response.status === 400 && Array.isArray(responseBody.invalidParams)) {
+    return {
+      errorMessage: responseBody.invalidParams
```

Comment:
Verwachten we 400 meldingen waar we de melding van willen doorzetten naar de eindgebruiker?